### PR TITLE
Clarify At-Most-One Semantic For DaemonSet Update

### PR DESCRIPTION
**Pull request by abhishekjiitr**
Make it explicit, that at most one pod will run on each node while updating a daemonset. As mentioned here: https://github.com/kubernetes/kubernetes/issues/48841

### CODE REVIEW

File: content/en/docs/tasks/manage-daemon/update-daemon-set.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ DaemonSet has two update strategy types:
 * RollingUpdate: This is the default update strategy.  
   With `RollingUpdate` update strategy, after you update a
   DaemonSet template, old DaemonSet pods will be killed, and new DaemonSet pods
-  will be created automatically, in a controlled fashion.
+  will be created automatically, in a controlled fashion. At most one pod will be running on each node during the whole update process.
```
Comment (sftim) [inline content/en/docs/tasks/manage-daemon/update-daemon-set.md]: I think it's worth pointing out that the “at most one” applies to Pods from the DaemonSet, and not to the set of Pods running via that Node's kubelet.

Comment (abhishekjiitr) [reply content/en/docs/tasks/manage-daemon/update-daemon-set.md]: clarified this in new commit

File: content/en/docs/tasks/manage-daemon/update-daemon-set.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ DaemonSet has two update strategy types:
 * RollingUpdate: This is the default update strategy.  
   With `RollingUpdate` update strategy, after you update a
   DaemonSet template, old DaemonSet pods will be killed, and new DaemonSet pods
-  will be created automatically, in a controlled fashion.
+  will be created automatically, in a controlled fashion. At most one pod of the daemonset will be running on each node during the whole update process.
```
Comment (sftim) [inline content/en/docs/tasks/manage-daemon/update-daemon-set.md]: a nit:
```suggestion
  will be created automatically, in a controlled fashion. At most one Pod of the DaemonSet will be running on each node during the whole update process.
```

Comment (abhishekjiitr) [reply content/en/docs/tasks/manage-daemon/update-daemon-set.md]: fixed

**Pull request merged**